app/views/assets/update.php

```diff
@@ -0,0 +1,175 @@
+                        <i class="glyphicon glyphicon-info-sign"></i>
+                        Help
+                    </a>
+                </li>
+            </ul>
+
+            <ul class="nav navbar-nav navbar-left">
+                <li>
+                    <a href="#menu-toggle" id="menu-toggle">
+                        Modules
+                    </a>
+                </li>
+            </ul>
+
+        </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+</nav>
+<body style="padding: 50px">
+<div id="wrapper" class="toggled">
+    <!-- Sidebar -->
+    <div id="sidebar-wrapper">
+        <ul class="sidebar-nav">
+
+            <li>
+                <a href="<?php echo base_url(); ?>missing/">
+                    Missing People
+                </a>
+            </li>
+
+            <li>
+                <a href="<?php echo base_url(); ?>organisations/">
+                    Organisations
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Camps
+                </a>
+            </li>
+
+            <li>
+                <a href="<?php echo base_url(); ?>assets/">
+                    Assets
+                </a>
+            </li>
+
+            <li>
+                <a href="<?php echo base_url(); ?>requests/">
+                    Requests
+                </a>
+            </li>
+
+        </ul>
+    </div>
+    <div id="page-content-wrapper" style="background-color: #ffffff;">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-12">
+                    <div class="page-header">
+                        <h1 style="text-align: center;">
+                            Update an Asset<br/>
+                        </h1>
+                        <br /><br />
+
+                        <?php
+                        if ($is_success !== null) {
+                            if ($is_success == true) {
+                                echo "<div class='alert alert-success'>Asset Updated</div>";
+                            }
+                            else {
+                                echo "<div class='alert alert-danger'>Asset not updated</div>";
+                            }
+                            echo "<br />";
+                        }
+                        ?>
+
+                        <form action='' method='POST'>
+                            <fieldset>
+                                <div class="panel panel-primary">
+                                    <div class="panel-heading" style="font-size: large; padding-bottom: 15px">
+                                        <b>Asset Details</b>
+                                        <div class="btn btn-info" id="show" style="float: right">Hide</div>
+                                    </div>
+
+                                    <input id="id" type="hidden" name="id" value="<?php echo $id ?>" required/><br>
```

Comment:
Move this input box out of this div, i.e., to **Line 143** because when you click on hide it does not hide properly, a blank div remains visible after clicking on hide also.